editor/src/components/canvas/controls/outline-control.tsx

```diff
@@ -208,6 +208,42 @@ export const OutlineControls = (props: OutlineControlsProps) => {
     props.selectedViews,
   ])
 
+  const parentOutlines = React.useMemo(() => {
+    return props.selectedViews.map((view) => {
+      const parentPath = EP.parentPath(view)
+      const parentElement = MetadataUtils.findElementByElementPath(
+        props.componentMetadata,
+        parentPath,
+      )
+      const parentFrame = MetadataUtils.getFrameInCanvasCoords(parentPath, props.componentMetadata)
+      if (
+        MetadataUtils.isFlexLayoutedContainer(parentElement) ||
+        MetadataUtils.isGridLayoutedContainer(parentElement)
+      ) {
+        if (parentFrame != null) {
+          return (
+            <div
+              style={{
+                position: 'absolute',
+                left: parentFrame.x + props.canvasOffset.x,
+                top: parentFrame.y + props.canvasOffset.y,
+                width: parentFrame.width,
+                height: parentFrame.height,
+                outline: `1px dotted ${colorTheme.primary.value}`,
```

Comment:
I know we're not overly concerned with zoomable canvas here yet, but we do need to find a way to make this work with zoom / scale, otherwise we're restricted to `boxShadow` here (which means no dotted outline), or need differential styling based on canvas zoom values.

I did some digging here. Previously I thought values of < 1px get rounded up to the nearest integer. That is still the case. But apparently if you set `zoom` on the parent, you _can_ use a value of <1px on the outline, and it'll render correctly!

![image](https://user-images.githubusercontent.com/2945037/128154232-9fe40f1a-7d30-4b88-b9ab-bb8a626eb5a4.png)


For zoom < 1 we can obviously scale this up

Reply:
fixed! 🔍